src/stats/boxplot.jl

```diff
@@ -27,6 +27,7 @@ The boxplot has 3 components:
 - `whiskerwidth`: multiplier of `width` for width of T's on whiskers, or
     `:match` to match `width`
 - `show_outliers`: show outliers as points
+- `dodge`: vector (length of data) of grouping variable to create multiple side-by-side boxes at the same `x` position
```

Comment:
Could be worth to mention that it has to be a vector of integers (at the moment there is no handling of categorical data). To close #1055 this should also document `dodge_gap`, but that can be done in a later PR